### PR TITLE
Fix issue #35 - replace Drupal region setting with custom block.

### PR DESCRIPTION
**Pull request by Graham-72**
I have written this PR and am now testing it.
Comments welcome.

### CODE REVIEW

File: config/rules.settings.json [hunk 1]
```diff
@@ -1,11 +1,10 @@
 {
   "_config_name": "rules.settings",
-  "rules_log_errors": "RulesLog::WARN",
+  "rules_log_errors": 3,
```
Comment (herbdool) [inline config/rules.settings.json]: This should probably stay "RulesLog::WARN", no? And there doesn't seem to be a reason for other small tweaks to this file. Except for rules_debig_region.

File: rules.module [hunk 2]
```diff
@@ -1327,44 +1327,6 @@ function rules_menu_add_element_title($array) {
...
-        '#markup' => $log,
-      );
-      $page[$region]['rules_debug']['#theme_wrappers'] = array('rules_log');
-    }
-    unset($_SESSION['rules_debug']);
-  }
-
-  if (rules_show_debug_output()) {
-    $region = rules_debug_log_region();
-    $page[$region]['rules_debug']['#pre_render'] = array('rules_debug_log_pre_render');
```
Comment (herbdool) [inline rules.module:1364]: Is there a prerender function that is no longer being used?

File: config/rules.settings.json [hunk 3]
```diff
@@ -1,11 +1,10 @@
...
   "rules_log_level": "self::INFO",
-  "rules_debug_log": "0",
-  "rules_debug": "FALSE",
-  "rules_debug_region": "",
-  "theme_default": "bartik",
+  "rules_debug_log": 0,
+  "rules_debug": 0,
+  "theme_default": "basis",
```
Comment (herbdool) [inline config/rules.settings.json]: The theme variables probably aren't needed. If they are shouldn't they just be called from the core config?

**Review by herbdool — CHANGES_REQUESTED** [review]
I haven't tested this yet so I'll do that soon. But I left a few comments.

File: config/rules.settings.json (hunk 1 above)
Comment (Graham-72) [reply config/rules.settings.json]: Yes, temporary changes accidentally included in this PR, my mistake.

File: rules.module (hunk 2 above)
Comment (Graham-72) [reply rules.module:1364]: Yes, `function rules_debug_log_pre_render() ` seems redundant now.

File: config/rules.settings.json (hunk 3 above)
Comment (Graham-72) [reply config/rules.settings.json]: I have made a separate issue for this - #59

**Comment by Graham-72** [comment]
Abandoning and closing this PR in favour of a new one - to be created.

**Pull request closed without merging**